esrally/client/factory.py

```diff
@@ -367,8 +367,16 @@ def cluster_distribution_version(hosts, client_options, client_factory=EsClientF
     if versions.is_serverless(version_build_flavor):
         # overwrite static serverless version number
         version_number = "serverless"
-        authentication_info = es.perform_request(method="GET", path="/_security/_authenticate")
-        serverless_operator = authentication_info.body.get("operator", False)
+
+        # determine operator status if security enabled, otherwise assume operator privileges
+        # pylint: disable=import-outside-toplevel
+        from elasticsearch.exceptions import ApiError
+
+        try:
+            authentication_info = es.perform_request(method="GET", path="/_security/_authenticate")
+            serverless_operator = authentication_info.body.get("operator", False)
+        except ApiError:
+            serverless_operator = False
```

Comment:
I think the explicit assignment is more readable than `pass`. Another option is using `contextlib.suppress`:

```suggestion
        with contextlib.suppress(ApiError):
            authentication_info = es.perform_request(method="GET", path="/_security/_authenticate")
            serverless_operator = authentication_info.body.get("operator", False)
```

Reply:
However, the comment above says "otherwise assume operator privileges", so should this be assigned to True?

Reply:
Good catch. That was my initial assumption which failed in testing. Code is correct, but comment requires a change. I like the brevity of `contextlib`, so let me use that as well :)